nomination/templates/nomination/url_add.html

```diff
@@ -8,6 +8,19 @@
 
 {% block title %}Nomination Tool: Add/Edit a Site{% endblock %}
 
+{% block localized_styles %}
+    {% if metadata_vals %}
+        {% for met,vals in metadata_vals %}
+            {% if met.form_type == "select" or met.form_type == "selectsingle" %}
+                {% ifchanged %}{% comment %}This is a bit of a hacky way to prevent grabbing slimselect multiple times.{% endcomment %}
```

Comment:
It is hard not to want to suggest something else when spurred on by a comment containing the word "hacky"... :grin: 

What if we add a templatetag that would allow us to make use of `form_types`:
```
@register.filter
def has_value(json_str, value):
    return value in json.loads(json_str).values()
```
so we could do:
```
{% if form_types|has_value:"select" or form_types|has_value:"selectsingle" %}
```

Reply:
I could do that. Alternatively I could just pass the list of field types as an additional context variable and leave a little more logic out of the templates.

Reply:
Guess it's the same amount of logic within the templates either way. I'll do the template tag.